config/jobs/testing/testing-trusted.yaml

```diff
@@ -82,7 +82,9 @@ postsubmits:
         - ./config/jobs/ # path to prow job definitions
         - ./config/testgrid/dashboards.yaml # path to testgrid config
         - jetstack # name of the directory containing our testgrid configs
-        - jetstack/test-infra
```

Comment:
This format was invalid - it should just be the name of the repo, not the `org/repo` 😄